trustfall_core/test_data/tests/valid_queries/fold_with_outputs_transform_and_filter_greater_than.graphql.ron

```diff
@@ -0,0 +1,19 @@
+
+TestGraphQLQuery (
+    schema_name: "numbers",
+    query: r#"
+{
+    Number(min: 30, max: 30) {
+        ... on Composite {
+            value @output
+
+            primeFactor @fold @transform(op: "count") @filter(op: ">", value: ["$min_primes"]) {
+                prime: value @output
+            }
+        }
+    }
+}"#,
+    arguments: {
+        "min_primes": Uint64(2)
+    },
```

Comment:
`> 2` here also requires fully evaluating the `@fold` because it requires seeing a third element, and the fold only has 3 elements. Consider making `min_primes` be 1 instead of 2 to make this a test of the optimization's behavior.